### PR TITLE
demisto-sdk logging-standardization

### PR DESCRIPTION
**Pull request by tkatzir**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ilaner** [comment]
Looks good. Please ask the release owner to merge the release content branch for this branch once Demisto-SDK (with logging) is released.

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "^3.8,<3.11"
 
 [tool.poetry.group.dev.dependencies]
-demisto-sdk = "1.10.6"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev="efbad5491db83783bd410dfe20f0b815945bef2b"}
```
Comment (ilaner) [inline pyproject.toml:12]: please revert

Comment (tkatzir) [reply pyproject.toml:12]: Of course. I'll revert pyproject.toml and poetry.lock

**Review by ilaner — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (tkatzir) [reply pyproject.toml]: Of course. Both files reverted.

**Pull request merged**